.kokoro/continuous/acceptance.cfg

```diff
@@ -29,7 +29,7 @@ env_vars: {
 
 env_vars: {
   key: "SECRET_MANAGER_KEYS"
-  value: ""
+  value: ".kokoro/universe_domain_test_creds.sh"
```

Comment:
The value that should be here instead is the secret that you want to get added to a file, i.e. `client-library-test-universe-domain-credential`